py/autoevals/moderation.py

```diff
@@ -0,0 +1,59 @@
+from braintrust_core.score import Score, Scorer
```

Comment:
make sure to export it from __init__.py

Reply:
Added - https://github.com/braintrustdata/autoevals/pull/63/commits/10a36d9785a7428d7c8db3a9385367d10282dae6#diff-7c64160f44380423dfe586bd06f8409eae28432f9efeea27a4207d23659a4daeR37